opencensus/trace/noopspan.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+from opencensus.trace.span_context import generate_span_id
+from opencensus.trace.tracers import base
+
+
+class NoopSpan(object):
```

Comment:
In java we call this BlankSpan, because people may use this even for a request that they don't want to trace.

Reply:
Changed NoopSpan to BlankSpan...